.github/workflows/test.yaml

```diff
@@ -99,7 +147,7 @@ jobs:
         echo "::set-output name=dir::$(pip cache dir)"
 
     - name: pip cache
-      uses: actions/cache@v2
+      uses: actions/cache@v3
```

Comment:
I don't know what's better about v3 than v2 but the number is bigger so I guess there must be something. :wink:

Reply:
While testing, the cache was timing out on Windows.

And I read there were some improvements to reduce timeout...so I gave it a try.
We will see if this is fixed.

If we continue to see cache timeout errors on Windows, then this is not fixed.